### DISCUSSION TITLE
No Emails, No Logs

### DISCUSSION THREAD

**Discussion by kj4awr** [hrfee/jfa-go#175] [Q&A]
I am running 0.3.9 version of jfa-go on 2 Windows 10 boxes and also Windows 11 box with the standalone version of Jellyfin, each of the 3 machines are on completely different networks, and I have tried to setup email using my custom domain, using a gmail account, using an outlook account and using a mailgun account I have not gotten a single email to send as they error out every time. I looked at open ports while sending and it appears that going out to various SMTP servers worked, but the emails were never sent. Nothing in spam, nothing in sent folders, nothing working at all. I have tried looking at the log file for jfa-go but it has sat at 0kb in size since I first opened jfa-go. I have run the app as admin, regular user, etc and still not one single line gets written to the logs. How can I troubleshoot this?

**Comment by roand0617** [comment]
Hello, I have Windows 11 and confirm the same problem, I cannot send emails from any provider (Gmail, Outlook) and when trying to check the log file it is empty, either opening it from the tray or from the temp folder. I appreciate your cooperation in reviewing this problem. I have version 0.3.9 of jfa-go.

**Comment by hrfee** [comment]
Sorry for the late reply, another user is also having issues accessing logs and i've just noticed that errors in opening the log file are being ignored. That should be fixed, could either of you try running the latest unstable version in the terminal? You can do this by Shift-Right clicking the folder the executable's stored in in your explorer window and pressing "Open Powershell window here", then typing `jfa-go.exe` and pressing Enter.
